lib/board.rb

```diff
@@ -1,37 +1,37 @@
 class Board
   attr_reader :cells
+
   def initialize(rows = 4, columns = 4)
     @number_range = number_range(columns)
     @letter_range = letter_range(rows)
     @cells = cell_generator(rows, columns)
   end
 
   def number_range(columns)
-    @number_range = ("1".."#{columns}").to_a
+    ("1".."#{columns}").to_a
```

Comment:
Good call! If it's returning that value already I guess we don't need to reassign the value of our variable.